process/smartContract/process.go

```diff
@@ -250,7 +250,7 @@ func (sc *scProcessor) doExecuteSmartContractTransaction(
 	snapshot := sc.accounts.JournalLen()
 
 	var vmInput *vmcommon.ContractCallInput
-	vmInput, err = sc.createVMCallInput(tx)
+	vmInput, err = sc.createVMCallInput(tx, txHash)
```

Comment:
This is not the original txhash which should be needed in case of async callback.use setOriginalTxHash function

Reply:
Correct. Replicated the behavior of `setOriginalTxHash()` in `sc.createVMCallInput()`.